mesh_segmenter/CMakeLists.txt

```diff
@@ -4,8 +4,17 @@ project(mesh_segmenter)
 ## Compile as C++14,
 add_compile_options(-std=c++14)
 
-find_package(VTK 7.1 REQUIRED NO_MODULE)
-include(${VTK_USE_FILE})
+## Try to get VTK 7.1.  If that fails, try to get VTK 8.2
+find_package(VTK 7.1 QUIET NO_MODULE)
+if(${VTK_FOUND})
```

Comment:
I think it my be better to remove the version from the find package and then check the package version to make sure it meets the correct version. The current state noether will only build with 7.1 and 8.2. This [link](https://stackoverflow.com/questions/16667017/cmake-express-the-greater-or-equal-statement) has information on how to check if the version is greater or equal.

PACKAGE_FIND_VERSION
    full requested version string
PACKAGE_FIND_VERSION_MAJOR
    major version if requested, else 0
PACKAGE_FIND_VERSION_MINOR
    minor version if requested, else 0
PACKAGE_FIND_VERSION_PATCH
    patch version if requested, else 0
PACKAGE_FIND_VERSION_TWEAK
    tweak version if requested, else 0
PACKAGE_FIND_VERSION_COUNT
    number of version components, 0 to 4